### PR TITLE
Update python version test coverage

### PR DESCRIPTION
**Pull request by shalomb**
* Support newer python3 versions
* Drop deprecated (at least on travis-ci) python versions

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +18,5 @@ after_success:
   - coveralls
 notifications:
   email:
-    recipients:
-      - yukino0131@me.com
     on_success: always
     on_failure: always
```
Comment (shalomb) [inline .travis.yml:21]: Had to drop this on my testing. Not sure if this is needed as this is the default on travis no?

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 python:
-  - "2.6"
   - "2.7"
-  - "3.3"
   - "3.4"
   - "3.5"
+  - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.9-dev"
```
Comment (shalomb) [inline .travis.yml]: Should perhaps drop this as `3.9-dev` will eventually go away after `3.9` is released.

Comment (shalomb) [reply .travis.yml]: Dropped in 47fc333

**Comment by ikegami-yukino** [comment]
@shalomb 
Thanks a lot!

**Pull request merged**